python/shotgun_desktop/startup.py

```diff
@@ -313,6 +313,18 @@ def _quit(self):
         """
         self.exit(self.CLOSE_APP)
 
+    def exec_(self):
+        """
+        Execute the local event loop. If CmdQ was hit in the past, it will be handled just as if the
+        user had picked the Quit menu.
```

Comment:
minor: may be worth adding a `:returns:` to the doc string documenting the return values.